### PR TITLE
refactor(@vtmn/react): allow theming for React

### PR DESCRIPTION
**Pull request by lauthieb**
## Pull Request checklist

🚨 Please review the [guidelines for contributing](../blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Don't request your main!
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] Check PR name, it must follow the https://www.conventionalcommits.org pattern
- [x] If it's a new component in CSS, ask for design review

## Description

Allow theming for react components/

Thanks a lot to @jleveugle for this PR.

## Does this introduce a breaking change?

- No

## Other information

❤️ Thank you!

### CODE REVIEW

File: packages/showcases/react/.storybook/preview-head.html [hunk 1]
```diff
@@ -8,7 +8,98 @@
...
 
 <style>
+  /* Extended toolbar */
+  .extended-toolbar {
+    background-color: var(--vtmn-semantic-color_background-primary);
+    padding: 1rem;
+    border-bottom: 1px solid var(--vtmn-semantic-color_border-secondary);
+    position: fixed;
+    width: 100%;
+    z-index: 99999;
```
Comment (GaspardMathon) [inline packages/showcases/react/.storybook/preview-head.html:18]:
```suggestion
    z-index: 10;
```

File: packages/showcases/react/.storybook/preview-head.html [hunk 2]
```diff
@@ -8,7 +8,98 @@
...
+    margin-bottom: 1rem;
+  }
+  .sb-show-main.sb-main-padded {
+    padding: 0;
+  }
+
+  /* Stories' preview */
+  #root {
+    padding: 1rem;
+    margin-top: 150px;
```
Comment (GaspardMathon) [inline packages/showcases/react/.storybook/preview-head.html:31]:
```suggestion
    margin-top: 180px;
```

**Review by GaspardMathon — CHANGES_REQUESTED** [review]
Need CSS changes

File: packages/showcases/react/.storybook/preview-head.html (hunk 2 above)
Comment (lauthieb) [reply packages/showcases/react/.storybook/preview-head.html:31]: Already fixed in https://github.com/Decathlon/vitamin-web/commit/cd8c64a1242619186ea73c32998cd3d706878396
The bug comes from the select

**Pull request merged**